frontend/src/components/navigation/menuItem.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Layout from 'antd'
+import "./index.css"
+import styled from "styled-components";
+import {ProfileFilled } from '@ant-design/icons'
+import './index.css'
+function MenuItem(){
```

Comment:
Trochę w inny sposób trzeba to ogarnąć. Zerknij sobie jak wygląda stylowanie przycisku w `src/components/buttons/Button.tsx`

W typ przypadku będzie to coś takiego:
`
import { MenuItem as AMenuItem } from 'antd';

const MenuItem = styled(AMenuItem)
`